lib/src/data/communication/api/api_exception.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:io';
+
+import 'package:dio/dio.dart';
+
+enum ApiExceptionType {
```

Comment:
jeżeli wykorzystywane jest w warstwie UI to powinno być zadeklarowane w warstwie domeny.

Reply:
nie będzie tego przy stworzeniu osobnej klasy dla danego typu błędu